src/main/java/ladder/game/Result.java

```diff
@@ -0,0 +1,20 @@
+package ladder.game;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Result {
```

Comment:
일급 컬렉션 👍 복수 명사를 클래스 이름으로 사용하면 컬렉션임을 더 쉽게 표현할 수 있겠죠?

```suggestion
public class Results {
```